src/Lykke.Job.TxDetector/Events/ConfirmationSavedEvent.cs

```diff
@@ -9,5 +10,9 @@ public class ConfirmationSavedEvent
         public string TransactionHash { get; set; }
         [ProtoMember(2)]
         public string ClientId { get; set; }
+        [ProtoMember(3)]
+        public IBlockchainTransaction BlockchainTransaction { get; set; }

```

Comment:
There is no need to sending full transaction DTO. The consumer can retrieve all needed information by transaction hash.